compiler/damlc/daml-compiler/src/DA/Daml/Compiler/DataDependencies.hs

```diff
@@ -108,6 +108,9 @@ generateSrcFromLf env = noLoc mod
 
     convDataCons :: T.Text -> LF.DataCons -> [LConDecl GhcPs]
     convDataCons dataTypeCon0 = \case
+            LF.DataSynonym _ ->
+              [] -- TODO(NICK) write the Haskell type synonym
```

Comment:
In theory, it will (usually? / always?) be a (DAML) typeclass actually. This is something I'm pretty unclear about at the moment  -- how we will be able to recover the typeclass definition from the synonym definition.